odoo/tools/float_utils.py

```diff
@@ -51,7 +53,8 @@ def float_round(value, precision_digits=None, precision_rounding=None, rounding_
     """
     rounding_factor = _float_check_precision(precision_digits=precision_digits,
                                              precision_rounding=precision_rounding)
-    if rounding_factor == 0 or value == 0: return 0.0
+    if value == 0:
```

Comment:
Is it really useful to remove the test on `rounding_factor` here? It should not be zero by construction but that's not obvious when reading only the code of `float_round`, and makes one wonder about the division by zero at the next operation.

Reply:
I only removed, because, as you said, by construction it shouldn't be zero. But I agree with you, is better if we leave it as is, to avoid misunderstandings.